### PR TITLE
[redfish] Add RackHD Manager

### PR DESCRIPTION
**Pull request by zyoung51**

### CODE REVIEW

**Comment by zyoung51** [comment]
Depends on: https://github.com/RackHD/RackHD/pull/223
@RackHD/corecommitters @uppalk1 @tannoa2 @keedya @jlongever

File: lib/api/redfish-1.0/managers.js [hunk 1]
```diff
@@ -34,42 +55,76 @@ var getManager = controller(function(req, res) {
...
+            var httpsLocation = _.find(locations, {httpsEnabled: true});
+
+            var services = [];
+            if(httpLocation) {
+                services.push({ name: 'HTTP', port: httpLocation.port });
+            }
+            if(httpsLocation) {
+                services.push({ name: 'HTTPS', port: httpsLocation.port });
+            }
+            services.push({ name: 'SSDP', port: 1900 });
```
Comment (jlongever) [inline lib/api/redfish-1.0/managers.js]: We'll need to followup with this and add disable/enable controls per Redfish spec.

**Comment by jlongever** [comment]
test this please

**Comment by jlongever** [comment]
test this please

**Comment by zyoung51** [comment]
test this please

**Comment by jlongever** [comment]
test this please

**Pull request merged**